docs/source/docs/explanation/formats/imagenet.md

```diff
@@ -39,38 +39,41 @@ Load ImageNet dataset through the Python API:
 import datumaro as dm
 
 dataset = dm.Dataset.import_from('<path_to_dataset>', format='imagenet_txt')
+# or
+dataset = dm.Dataset.import_from('<path_to_dataset>', format='imagent')
 ```
 
 For successful importing of ImageNet dataset the input directory with dataset
 should has the following structure:
 
-<!--lint disable fenced-code-flag-->
-{{< tabpane >}}
-  {{< tab header="imagenet">}}
-imagenet_dataset/
-├── label_0
-│   ├── <image_name_0>.jpg
-│   ├── <image_name_1>.jpg
-│   ├── <image_name_2>.jpg
-│   ├── ...
-├── label_1
-│    ├── <image_name_0>.jpg
-│    ├── <image_name_1>.jpg
-│    ├── <image_name_2>.jpg
-│    ├── ...
-├── ...
-  {{< /tab >}}
-  {{< tab header="imagenet_txt">}}
-imagenet_txt_dataset/
-├── images # directory with images
-│   ├── <image_name_0>.jpg
-│   ├── <image_name_1>.jpg
-│   ├── <image_name_2>.jpg
-│   ├── ...
-├── synsets.txt # optional, list of labels
-└── train.txt   # list of pairs (image_name, label)
-  {{< /tab >}}
-{{< /tabpane >}}
+- `imagenet` format:
+  <!--lint disable fenced-code-flag-->
+  ```
+  imagenet_dataset/
+  ├── label_0
+  │   ├── <image_name_0>.jpg
+  │   ├── <image_name_1>.jpg
+  │   ├── <image_name_2>.jpg
+  │   ├── ...
+  ├── label_1
+  │    ├── <image_name_0>.jpg
+  │    ├── <image_name_1>.jpg
+  │    ├── <image_name_2>.jpg
+  │    ├── ...
```

Comment:
These lines seem indented by 1 space compared to others.

Reply:
Fixed in https://github.com/openvinotoolkit/datumaro/pull/985/commits/8e5ebb817a14b8e73fac486a1171bbac5e8dc740